content/docs/1.1.0/advanced-resources/longhorn-client-python.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Longhorn Client Python
+weight: 5
+---
+
+Currently, you can operate Longhorn using Longhorn UI.
+We are planning to build a dedicated Longhorn CLI in the future.
+While waiting for the CLI, we provide you a Python tool, Longhorn Client Python, for automation/scripting purposes.
```

Comment:
`In the meantime, you can access Longhorn API using Python binding, as we demonstrated below.`

Reply:
Thank you. Fixed